base/lock.jl

```diff
@@ -496,6 +497,9 @@ This provides an acquire & release memory ordering on notify/wait.
 
 !!! compat "Julia 1.8"
     The `autoreset` functionality and memory ordering guarantee requires at least Julia 1.8.
+
+!!! compat "Julia 1.12"
+    `isopen(::Event)` requires at least Julia 1.12.
 """
```

Comment:
This compat notice should be part of an `Event`-specific docstring on the `isopen` definition below. The generic `isopen` has a docstring that isn't really applicable to `Event`.

Reply:
Right, on second thoughts this is actually a confusing implementation since `Event` can produce multiple events with `autoreset=True`. Changed the implementation in 70f719e38f to match `isopen()` better.